src/dbnode/integration/write_quorum_test.go

```diff
@@ -231,12 +230,15 @@ func makeTestWrite(
 	require.NoError(t, err)
 
 	nspaces := []namespace.Metadata{md}
-	nodes, topoInit, closeFn := newNodes(t, numShards, instances, nspaces, false,
-		bootstrapper.NoOpAllBootstrapperName)
+	nodes, topoInit, closeFn := newNodes(t, numShards, instances, nspaces, false)
 	now := nodes[0].NowFn()()
 
 	for _, node := range nodes {
 		node.SetOpts(node.Opts().SetNumShards(numShards))
+		for _, ns := range node.Namespaces() {
+			// write empty data files to disk so nodes could bootstrap
+			writeTestDataToDisk(ns, node, generate.SeriesBlocksByStart{}, 0)
```

Comment:
This is needed because `UninitializedTopology` would not work here? 
I'm afraid this workaround might become what `NoOpAllBootstrapper` was, let's discuss this offline.